src/components/elementView/elementTab.tsx

```diff
@@ -465,96 +514,64 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
                 minimal
               />
             ) : null}
-            <Alert
-              cancelButtonText="Cancel"
-              className={Classes.DARK}
-              intent={Intent.PRIMARY}
-              confirmButtonText="Confirm Export"
-              isOpen={this.state.fileNameIsOpen}
-              onCancel={() => {
+            <Dialog
```

Comment:
i had the sudden urge to beautify this alert so i made it into a dialog, which gives u title and an x out button